File: api/Controllers/Models/ViewModels/OrderLineItemVM.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace IPKP___API.Controllers.Models.ViewModels
{
    public class OrderLineItemVM
    {
        //delivery request
        public double Delivery_Price { get; set; }
        public string Delivery_Company_Name { get; set; }
        public string Delivery_Status { get; set; }
        public string StreetName { get; set; }
        public int StreetNumber { get; set; }
        public string Dwelling_Type { get; set; }
        public int Unit_Number { get; set; }
        public string City { get; set; }
        public string Province { get; set; }
        public string AreaCode { get; set; }

        //orderrequest
        public Guid Customer_ID { get; set; }
        public string Customer_UserName { get; set; }
        public DateTime Order_Request_Date { get; set; }
        public double Order_Request_Total_Price { get; set; }
        public string Order_Status { get; set; }

        //Personalisation design 
        public string Image_File { get; set; }
        public double Image_Price_Amount { get; set; }

        public string Design_Text { get; set; }
        public double Text_Price_Amount { get; set; }

        public string Stock_Item_Name { get; set; }
        public decimal Stock_Item_Price { get; set; }
        public string Stock_Item_Size { get; set; }
        public string Stock_Colour_Name { get; set; }
        public double Personalisation_Design_Price { get; set; }

        //order line item 
        public Guid Order_Line_Item_ID { get; set; }
        public double Order_Line_Item_Price { get; set; }
        public int Order_Line_Item_Quantity { get; set; }
        public double Order_Line_Item_Total_Price { get; set; }
    }
}

File: mobileapp/mobileapp/src/app/ViewModels/orderlineitemVM.ts
export class OrderLineItemVM{
    delivery_Price!: number
    delivery_Company_Name!: string
    delivery_Status!: string
    streetName!: string
    streetNumber!: number
    dwelling_Type!: string
    unit_Number!: number
    city!: string
    province!: string
    areaCode!: string
   
    customer_ID!: number
    customer_Username!: string

    order_Request_Date!: Date
    order_Request_Total_Price!: number

    image_File!: string
    image_Price_amount!: number
    
    design_Text!: string
    text_Price_Amount!: number

    stock_Item_Name!: string
    stock_Item_Price!: number
    stock_Item_Size!: string
    stock_Colour_Name!: string
    personalisation_Design_Price!: number

    order_Line_Item_ID!: number
    order_Line_Item_Price!: number
    order_Line_Item_Quantity!: number
    order_Line_Item_Total_Price!: number
    order_Status!: string
}